api/python/t4/snapshots.py

```diff
@@ -160,6 +160,46 @@ def hash_file(readable_file):
 
     return hasher.hexdigest()
 
+def bytes_to_int(input_bytes):
+    result = 0
+    for byte in input_bytes:
+        result = result * 256 + int(byte)
+    return result
+
+HASH_MAX = 2 ** 256
+
+def sum_hash(current, add_bytes):
+    """
+    Sums hashes for order-independence.
+
+    Args:
+        current(int)
+        add_bytes(bytes)
+
+    Returns:
+        New total
+    """
+    total = current + bytes_to_int(add_bytes)
+    total = total % HASH_MAX
```

Comment:
Why not just XOR the bit arrays instead of having all this arithmetic. They're both insecure ways of combining the hashes, so if you looking for an MVP might as well do that.